src/browser_authentication_app_skeleton/src/pages/auth_layout.cr

```diff
@@ -1,15 +1,25 @@
 abstract class AuthLayout
-  # Edit shared layout code in src/components/shared/layout.cr
-  include Shared::Layout
+  include Lucky::HTMLPage
+
+  abstract def content
+  abstract def page_title
+
+  # The default page title. It is passed to `Shared::LayoutHead`.
+  #
+  # Add a `page_title` method to pages to override it. You can also remove
+  # This method so every page is required to have its own page title.
+  def page_title
+    "Welcome"
+  end
 
   def render
     html_doctype
 
     html lang: "en" do
-      shared_layout_head
+      mount Shared::LayoutHead.new(page_title: page_title, context: @context)
 
       body do
-        render_flash
+        mount Shared::FlashMessages.new(@context.flash)
```

Comment:
Uses `new` so you can do all kinds of stuff with the class before it is rendered/mounted